.github/workflows/ci.yaml

```diff
@@ -29,3 +29,19 @@ jobs:
         [
           {"folder": ".", "bzlmodEnabled": false}
         ]
+  # For branch protection settings, this job provides a "stable" name that can be used to gate PR merges
+  # on "all matrix jobs were successful".
+  conclusion:
+    needs: test
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - uses: technote-space/workflow-conclusion-action@45ce8e0eb155657ab8ccf346ade734257fd196a5 # v3.0.3
```

Comment:
It seems this action's repo has been archived - do we want to be depending on a un-maintained action here? There a lot of active forks of this action though.

Reply:
I noticed that as well, but I don't have an objection. If the code is working, it doesn't need a maintainer. If it's broken, we'll investigate and switch to a fork if needed.